index.html

```diff
@@ -6,19 +6,25 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Portfolio Website 2</title>
     <link rel="stylesheet" href="style.css">
-   <link rel='stylesheet' href='https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css'>  
-   <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>                                   
+   <link rel='stylesheet' href='https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css'>                              
 </head>
 <body>
     <!-- header section -->
     <header>
-        <div class="logo">MY LOGO</div>
+        <div class="logo">My Logo</div>
         <div class="menu">&#9776;</div>
+        <div class="Desktop_menu">
+            <ul>
+                <li>Portfolio</li>
+                <li>About</li>
+                <li>Contact</li>
+            </ul>
+        </div>
```

Comment:
Great job on the desktop navbar! I wanted to offer **a couple of friendly suggestions to further enhance its appearance** and align it closely with the provided [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=39-122):

- [ ] Would you mind checking the font-weight of your links in the desktop navbar? 👌
- [ ] **[OPTIONAL]** It would be fantastic if you **could consider adding your links within a `nav` element** for a more improved HTML structure. 🤓

   The reason for these suggestions is that adjusting the font-weight of the links can help ensure a consistent visual style throughout the navbar. Additionally, **organizing the links within a nav element provides a better semantic structure, which can contribute to improved accessibility and maintainability.**

Reply:
ok